docs/src/main/sphinx/connector/jdbc-procedures.fragment

```diff
@@ -9,3 +9,10 @@
   USE example.example_schema;
   CALL system.flush_metadata_cache();
   ```
+
+- `system.update('query')`
+
+  ```sql
+  USE example.example_schema;
+  CALL system.update(query => 'ALTER TABLE your_table ALTER COLUMN your_column DROP DEFAULT')
```

Comment:
Is it specific to `ALTER` or does it support `CREATE` syntax as well ?

Reply:
It supports any syntax that JDBC driver's `Statement#executeUpdate` method accepts.

Reply:
Is there a generic way to describe what that is? Or is that different for each JDBC driver. Can we link to the docs for each JDBC driver in the longer run then?